simdpp/dispatch/get_arch_string_list.h

```diff
@@ -84,10 +84,10 @@ inline Arch get_arch_string_list(const char* const strings[], int count, const c
     return res;
 #endif
 
-    int prefixlen = std::strlen(prefix);
-    for (int i = 0; i < count; ++i) {
+    auto prefixlen = std::strlen(prefix);
```

Comment:
I'd prefer the explicit type for the simple numeric types. Auto makes the code less readable in these cases.

Reply:
OK use size_t instead